### ISSUE TITLE
Option to disable certain commands

### ISSUE THREAD

**Issue by mntdamania** [kemayo/sublime-text-git#470]
Is it possible to disable certain commands such as `git reset --hard`. I just did this and was navigating quickly through the menu :cry:

**Comment by Pendrokar** [comment]
Yes, by commenting out commands in the Default.sublime-commands file.

**Comment by kemayo** [comment]
There's not. That said, I'll put confirmation-dialogs on my todo list.
